### PR TITLE
add LICENSE and CONTRIBUTING documents

### PR DESCRIPTION
**Pull request by pkarman**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+## Welcome!
+
+We're so glad you're thinking about contributing to an 18F open source project! If you're unsure about anything, just ask -- or submit the issue or pull request anyway. The worst that can happen is you'll be politely asked to change something. We love all friendly contributions.
```
Comment (sbc100) [inline CONTRIBUTING.md]: Word wrap?   (I hate long line, sorry)  Is this also part of the 18f template?  Perhaps we need fix it there too?

Comment (pkarman) [inline CONTRIBUTING.md]: You may find folks with equally strong feelings about inserting newline characters. I am agnostic.

These files are pulled directly from the 18F templates.

Comment (jgrevich) [inline CONTRIBUTING.md]: I vote for WW. Then the text is also formatted for people viewing these files in a terminal. It does not affect the formatting in GH which ignores WW, for example see https://github.com/18F/identity-sp-rails/blob/license-contributing/LICENSE.md vs https://raw.githubusercontent.com/18F/identity-sp-rails/license-contributing/LICENSE.md

Comment (jgrevich) [inline CONTRIBUTING.md]: That said, we should probably update the ['canonical' source](https://github.com/18F/open-source-policy) for these files.

Comment (pkarman) [inline CONTRIBUTING.md]: I'm not particularly interested in fighting the fight upstream for the canonical templates.

Happy to change it here per the 2 votes given. Done in 1c41f57

**Pull request merged**